queue_job/delay.py

```diff
@@ -493,7 +493,10 @@ def _tail(self):
 
     def __repr__(self):
         return "Delayable({}.{}({}, {}))".format(
-            self.recordset, self._job_method.__name__, self._job_args, self._job_kwargs
+            str(self.recordset),
+            self._job_method and self._job_method.__name__ or "",
```

Comment:
```suggestion
            self._job_method.__name__ if self._job_method else "",
```

`... and ... or ...` is a pattern I prefer to avoid